packages/graphiql/src/components/Tabs.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+function TabCloseButton(props: { onClick: () => void }) {
+  return (
+    <button
+      className="close"
+      aria-label="Close Tab"
+      onClick={ev => {
+        ev.stopPropagation();
+        props.onClick();
+      }}
+    />
+  );
+}
+
+export function Tab(props: {
+  isActive: boolean;
+  title: string;
+  isCloseable: boolean;
+  onSelect: () => void;
+  onClose: () => void;
+}): React.ReactElement {
+  return (
+    <div
+      role="button"
```

Comment:
I think we need to use `tab` role here:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role

Reply:
it seems aria `tablist` spec also describes what the keyboard shortcuts should be, so that clears all of that up!

Reply:
Good points, I just added the 3!